### DISCUSSION TITLE
Ongoing stat issues

### DISCUSSION THREAD

**Discussion by mdryden** [110yards/fantasy#275] [General]
This week the system did not calculate the scores properly at the end of the week.  This issue has been fixed and scores/results should be correct for week 1 now.

However, in applying this fix, I missed some things:

1) I forgot to set the week back to week 1.  By allowing the week to stay on week 2, rosters were left unlocked for several hours today.  This means that managers were able to shuffle players around their roster.

2) I forgot to disable waivers.  This means that managers were allowed to place waiver claims during that time as well.

Two issues have occurred as a result:

1) When I re-ran the end of week process after applying my fix, it incorrectly ran waivers instead of re-calculating results.

2) Then when I fixed that and re-ran the calculations, the calculations were run on the current state of the rosters, which is not necessarily what they were as of the end of week 1 (when they were still locked).

In short, this means that this week's results may be wrong for your league.  Managers may have received more or less points for the week than they should have, because they changed their roster before I made the fix (which is my fault, to be clear, not theirs).  And players may have been awarded on waivers way too early (and those players may have been included in the week 1 results).

I'm sorry to say that I don't have a good way to fix this issue across the board, but what I can do is help you out on an individual basic.

If your league has been negatively affected by this and you would like me to fix your results manually, either by undoing waiver claims or adjusting results, please reach out.  Discord is best but I will monitor this thread as well.  If you can live with the week 1 outcome as they stand, I appreciate that.

Sorry for the error.

---
Update 1: Late update on this feature - it was rolled back because there was a major bug with it.

---
Update 2: it has been turned back on.

I have discovered that there are some issues in the source data I am using and I hope that this feature can tide over leagues until I can come up with a better solution. Discord is the best place for up to the minute updates, but I will try to continue updating this thread as well.

**Comment by CALGARYCALROSS** [comment]
I have Cam Echols-Luper on my roster and he got zero points. He is in the list twice. Just Cam Echols is the other name. Look like the shorter name got points.

**Reply by mdryden** [reply]
Update: I've decided that instead of doing a bunch of manual fixes, a better approach is to give commissioners the power to apply a manual adjustment to past results.  This will allow leagues to resolve this issue, and to correct issues in the future without needing my help.

I hope to release this tonight or tomorrow.

**Comment by mdryden** [comment]
Late update on this feature - it was rolled back because there was a major bug with it.

More current update: it has been turned back on.

I have discovered that there are some issues in the source data I am using and I hope that this feature can tide over leagues until I can come up with a better solution.  Discord is the best place for up to the minute updates, but I will try to continue updating this thread as well.
